### PR TITLE
Pancake sort implemented in Dart - #2151 resolved

### LINKED ISSUE #2151: Pancake Sort
- [x] C
- [x] C++
- [x] C# - @himanshusharma89
- [ ] CoffeeScript
- [ ] Dart - @Rukmini-Meda
- [x] Go - @KamalDGRT
- [ ] Java - @parjanyaacoder
- [x] JS - @sejalshri
- [ ] Kotlin - @sandhyabhan
- [ ] Php - @sakshikhachane
- [x] Python
- [ ] Readme - @Tawishi
- [x] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by Rukmini-Meda**
Fixes #2151 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Pancake sort is implemented in Dart

#### Languages Used:
- Dart

#### Files Added:
- Pancake_Sort.dart

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Pancake_Sort/Pancake_Sort.dart [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    print("Enter array elements:");
+    input = stdin.readLineSync();
+    var lis = input.split(" ");
+    List array = lis.map(int.parse).toList();
+
+    pancakeSort(array, n);
+
+    print(array.join(" "));
+}
+/**
```
Comment (MastersAbh) [inline Pancake_Sort/Pancake_Sort.dart:62]: give one blank line before sample

File: Pancake_Sort/Pancake_Sort.dart [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+void pancakeSort(List array, int n){
+    for(int i = n; i > 1; i --){
+        int max_index = findMax(array, i);
+        if(max_index != i - 1){
+            flip(array, max_index);
+            flip(array, i - 1);
+        }
+    }
+}
+// Driver method of the program
```
Comment (MastersAbh) [inline Pancake_Sort/Pancake_Sort.dart:45]: give one blank line before this line. and subsequent blank lines are not needed after the comment.

**Comment by Rukmini-Meda** [comment]
@MastersAbh I have made changes as you suggested and squashed the commits. Please merge

**Comment by Rukmini-Meda** [comment]
@MastersAbh Please merge this PR

**Pull request merged**